app/build.gradle

```diff
@@ -92,9 +89,10 @@ dependencies {
 }
 
 buildscript {
-    ext.kotlin_version = '1.1.1'
+    ext.kotlin_version = '1.1.2-3'
     repositories {
         jcenter()
+        maven { url 'https://dl.bintray.com/kotlin/kotlin-eap-1.1' }	
```

Comment:
This is needed for the EAP version of Kotlin, EAP was needed to make alpha work with 2.4.0-alpha7. But those changes have since been released in a stable version of Kotlin (1.1.2), so not needed anymore.